src/http_client_builder.rs

```diff
@@ -0,0 +1,15 @@
+use reqwest::{
+    header::{self, HeaderMap, HeaderValue},
+    ClientBuilder,
+};
+
+pub struct TestClientBuilder;
```

Comment:
This doesn't look like it's being used, so I suggest we remove it and keep using the `test_http_client` function.